src/components/Alert/common.ts

```diff
@@ -17,9 +19,20 @@ export interface StatefulAlertProps {
     onDismiss?: Rivet.Action
 }
 
+export const statefulPropTypes = {
+    variant: PropTypes.oneOf(['info', 'error', 'message', 'success']).isRequired,
```

Comment:
Since you took the time to alphabetize the enum type we might as well have it alphabetized here as well.